ui/app/AppLayouts/Chat/popups/GroupInfoPopup.qml

```diff
@@ -285,154 +193,106 @@ ModalPopup {
-
-                StyledText {
-                    text: !model.userName.endsWith(".eth") && !!contactRow.nickname ?
-                              contactRow.nickname : Utils.removeStatusEns(model.userName)
-                    anchors.left: identicon.right
-                    anchors.leftMargin: Style.current.padding
-                    anchors.verticalCenter: parent.verticalCenter
-                    font.pixelSize: 17
-
-                    StyledText {
-                        visible: model.publicKey === userProfile.pubKey
-                        anchors.left: parent.right
-                        anchors.leftMargin: 5
-                        //% "(You)"
-                        text: qsTrId("-you-")
-                        color: Style.current.secondaryText
-                        font.pixelSize: parent.font.pixelSize
-                    }
-                    MouseArea {
-                        anchors.fill: parent
-                        cursorShape: Qt.PointingHandCursor
-                        onClicked: {
-                            const userProfileImage = appMain.getProfileImage(model.publicKey)
-                            openProfilePopup(model.userName, model.publicKey, userProfileImage || model.identicon, '', contactRow.nickname, popup)
-                        }
-                    }
-                }
-
-                StyledText {
-                    id: adminLabel
-                    visible: model.isAdmin
-                    //% "Admin"
-                    text: qsTrId("group-chat-admin")
-                    anchors.right: parent.right
-                    anchors.verticalCenter: parent.verticalCenter
-                    font.pixelSize: 13
-                    color: Style.current.secondaryText
-                }
-
-                StyledText {
-                    id: moreActionsBtn
-                    visible: !model.isAdmin && popup.isAdmin
-                    text: "..."
-                    anchors.right: parent.right
-                    anchors.rightMargin: Style.current.smallPadding
-                    anchors.verticalCenter: parent.verticalCenter
-                    font.pixelSize: 20
-                    font.bold: true
-                    color: Style.current.secondaryText
-                    MouseArea {
-                        anchors.fill: parent
+                title: !model.userName.endsWith(".eth") && !!contactRow.nickname ?
+                           contactRow.nickname : Utils.removeStatusEns(model.userName)
+                //% "(You)"
+                titleAsideText: model.publicKey === userProfile.pubKey ? qsTrId("-you-") : ""
+                //% "Admin"
+                statusListItemTitle.font.pixelSize: 17
+                statusListItemTitleAside.font.pixelSize: 17
+                label: model.isAdmin ? qsTrId("group-chat-admin"): ""
+                image.source: appMain.getProfileImage(model.publicKey)|| model.identicon
+                image.isIdenticon: true
```

Comment:
`image.isIdenticon` can possibly be `false` no? Since this is the group member list, there's chances we want to render profile images here (which are not identicons)...

Reply:
`userProfileImage` seems to exist for this here